Jenkinsfile.template

```diff
@@ -19,28 +22,38 @@ pipeline {
                                scmSkip(deleteBuild: true, skipPattern:'.*\\[ci skip\\].*')
                             }
                         }
-                        echo "NODE_NAME = ${env.NODE_NAME}"
-                        sh 'podman build -t localhost/$IMAGE_NAME --pull --force-rm --no-cache .'
+                        container('podman') {
+                            echo "NODE_NAME = ${env.NODE_NAME}"
```

Comment:
The echo should probably be outside of the container block, but I messed this up on the rstudio-base and it doesn't seem to have harmed anything.